plugins/modules/pfsense_aggregate.py

```diff
@@ -144,6 +144,11 @@
         description: Blocks traffic from reserved IP addresses (but not RFC 1918) or not yet assigned by IANA.
         required: false
         type: bool
+      slaacusev4iface:
+        description: IPv6 will use the IPv4 connectivity link (PPPoE). Only used when dhcp6_type is slaac.
```

Comment:
Shouldn't this say `ipv6_type`, rather than ~`dhcp6_type`~?

Reply:
Yes, good catch.